chapters/en/chapter6/fine-tuning.mdx

```diff
@@ -0,0 +1,563 @@
+# Fine-tuning SpeechT5
```

Comment:
Ah it's a shame we won't have MMS fine-tuning ready in time, I think for safety we should stick with SpeechT5 fine-tuning since we know for sure that it works

Reply:
We can't fine-tune MMS checkpoints anyway, no? All of them are single-speaker AFAIR.

Reply:
@Vaibhavs10 AFAIK, we can't finetune MMS for TTS (we can for ASR). At least as of now.